Add a juliadocs ref to the "trailing one" rule

HenriLaurie opened the pull request:
I was surprised (perhaps shouldn't have been ...) that if A[1] is a valid index into A, so is A[1,1], and struggled to find the explanation in the docs

HenriLaurie committed:
Add a juliadocs ref to the "trailing one" rule

I was surprised (perhaps shouldn't have been ...) that if A[1] is a valid index into A, so is A[1,1], and struggled to find the explanation in the docs

JeffBezanson commented:
Thanks, and welcome! Seems like a good addition.

JeffBezanson commented on doc/src/manual/faq.md:
```diff
@@ -818,9 +818,9 @@ some ideas that might help to understand Julia's definition.
-  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them
-  as for all other array dimensionalities, so you can indeed index them as
-  `A[1]`, `A[1,1]`, etc.
+  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them as for all other array dimensionalities, 
```
Please remove the trailing whitespace.

HenriLaurie committed:
Tried to remove white space

Not sure I found any while space, I did change the line breaks a bit, hope that's right

fredrikekre commented on doc/src/manual/faq.md:
```diff
@@ -818,9 +818,9 @@ some ideas that might help to understand Julia's definition.
-  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them
-  as for all other array dimensionalities, so you can indeed index them as
-  `A[1]`, `A[1,1]`, etc.
+  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them 
```
```suggestion
  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them
```

mbauman replied:
This suggestion still needs to be committed (and is why Travis & AppVeyor are still failing).

fredrikekre commented on doc/src/manual/faq.md:
```diff
@@ -818,9 +818,9 @@ some ideas that might help to understand Julia's definition.
-  as for all other array dimensionalities, so you can indeed index them as
-  `A[1]`, `A[1,1]`, etc.
+  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them 
+  as for all other array dimensionalities, so you can indeed index them as `A[1]`, `A[1,1]`, etc; see 
```
```suggestion
  as for all other array dimensionalities, so you can indeed index them as `A[1]`, `A[1,1]`, etc; see
```

fredrikekre commented on doc/src/manual/faq.md:
```diff
@@ -818,9 +818,9 @@ some ideas that might help to understand Julia's definition.
-  `A[1]`, `A[1,1]`, etc.
+  index -- they’re just `A[]`. We can apply the same "trailing one" rule for them 
+  as for all other array dimensionalities, so you can indeed index them as `A[1]`, `A[1,1]`, etc; see 
+  [Omitted and extra indices](https://docs.julialang.org/en/v1/manual/arrays/#Omitted-and-extra-indices-1).
```
```suggestion
  [Omitted and extra indices](@ref).
```

mbauman commented:
This looks good to me with Fredrik's suggestions — but I'm unable to commit them to your fork myself.  Could you merge those suggestions?

HenriLaurie committed:
Update doc/src/manual/faq.md

Accepting frederikre's changes

Co-Authored-By: Fredrik Ekre <ekrefredrik@gmail.com>

HenriLaurie committed:
Update doc/src/manual/faq.md

removed trailing whitespace

Co-Authored-By: Fredrik Ekre <ekrefredrik@gmail.com>

HenriLaurie commented:
@fredrikekre --- Not sure why Travis CI and AppVeyor are failing, but the Details tab suggests it is the removal of whitespace and they name your code. I'm a bit out of my depth here, this is my first PR and I don't quite know how to interpret the messages or what to do next.

fredrikekre commented:
The problem is https://github.com/JuliaLang/julia/pull/32078#discussion_r285890686

mbauman commented:
In the future, if you leave this checkbox ticked then we'd be able to commit these minor sorts of fixes directly.

![Screen Shot 2019-05-24 at 12 17 54 PM](https://user-images.githubusercontent.com/154641/58345442-fb7dfc80-7e1d-11e9-8b50-e2a014b7c930.png)

HenriLaurie committed:
Another try to commit the suggestion

Co-Authored-By: Fredrik Ekre <ekrefredrik@gmail.com>

HenriLaurie commented:
Augh!! Still fails. 
Thanks @mbauman for the tip about not disallowing maintainers, will do that in future.
But I'd love more pointers about what to do now, I thought I had committed @fredrikekre removal of whitespace.

fredrikekre commented:
The failures are unrelated to your change. Lets just wait for the remaining jobs and then merge.
